### PR TITLE
touch panning

### PR DESCRIPTION
**Pull request by merceavanessa**

### CODE REVIEW

File: examples/index.html [hunk 1]
```diff
@@ -54,6 +54,7 @@
             <li><a href="selection/index.html">Rectangular selection support and zooming</a> and <a href="zooming/index.html">zooming with overview</a> (both with selection plugin)</li>
             <li><a href="interacting/index.html">Interacting with the data points</a></li>
             <li><a href="navigate/index.html">Panning and zooming</a> (with navigation plugin)</li>
+              <li><a href="navigateTouch/index.html">Moving Touch</a> (with navigation plugin)</li>
```
Comment (cipix2000) [inline examples/index.html]: Panning with touch

File: karma.conf.js [hunk 2]
```diff
@@ -39,6 +40,8 @@ module.exports = function(config) {
         // list of files / patterns to load in the browser
         files: sources.concat([
             'node_modules/phantomjs-polyfill-find/find-polyfill.js',
+            //'node_modules/hammerjs/hammer.min.js',
```
Comment (cipix2000) [inline karma.conf.js]: remove these

File: karma.conf.js [hunk 3]
```diff
@@ -39,6 +40,8 @@ module.exports = function(config) {
...
+            //'node_modules/hammer-touchemulator/touch-emulator.js',
```
Comment (cristiingineru) [inline karma.conf.js]: remove

File: package.json [hunk 4]
```diff
@@ -19,6 +19,7 @@
   "homepage": "https://github.com/ni-kismet/engineering-flot#readme",
   "devDependencies": {
     "concat": "^1.0.3",
+    "eslint": "^4.1.1",
```
Comment (cipix2000) [inline package.json]: remove this. eslint is  requested and configured in webcharts-development-settings repo.

File: jquery.flot.touchNavigate.js [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+1 is 100% (i.e. no change), 1.5 is 150% (zoom in), 0.7 is 70% (zoom out). You
+can set the default in the options. */
+
+/* global jQuery */
+
+(function($) {
+    'use strict';
+
+    var options = {
+        pan: {
```
Comment (cipix2000) [inline jquery.flot.touchNavigate.js]: remove this as it is already added by the navigation plugin

File: tests/jquery.flot.navigate.Test.js [hunk 6]
```diff
@@ -278,6 +279,66 @@ describe("flot navigate plugin", function () {
...
+
+        expect(xaxis.min).toBeCloseTo(-1,2);
+        expect(yaxis.max).toBeCloseTo(9,2);
+        expect(xaxis.max).toBeCloseTo(9,2);
+        expect(yaxis.min).toBeCloseTo(-1,2);
+
+      });
+    });
+
+  function sendTouchEvent(x, y, element, eventType) {
```
Comment (cristiingineru) [inline tests/jquery.flot.navigate.Test.js]: cleanup this function

File: package.json (hunk 4 above)
Comment (cristiingineru) [inline package.json]: ?

File: tests/jquery.flot.navigate.Test.js [hunk 7]
```diff
@@ -278,6 +279,66 @@ describe("flot navigate plugin", function () {
         });
     });
 
+    describe('touchDrag', function() {
+      it('should drag the plot (start + move + end)',function() {
```
Comment (cristiingineru) [inline tests/jquery.flot.navigate.Test.js]: Repeat this test for a different frameRate. You will need https://jasmine.github.io/2.0/introduction.html#section-Mocking_the_JavaScript_Timeout_Functions 

OR

Delete everything related to the framerate and add it back later.

File: jquery.flot.touchNavigate.js [hunk 8]
```diff
@@ -0,0 +1,158 @@
+/* Flot plugin for adding the ability to pan the plot.
```
Comment (cristiingineru) [inline jquery.flot.touchNavigate.js]: Cleanup or remove these misleading comments.

File: tests/jquery.flot.navigate.Test.js [hunk 9]
```diff
@@ -278,6 +280,66 @@ describe("flot navigate plugin", function () {
...
+      {
+        identifier: Date.now(),
+        target: element,
+        clientX: x,
+        clientY: y,
+        radiusX: 2.5,
+        radiusY: 2.5,
+        rotationAngle: 10,
+        force: 0.5,
+      };//);
```
Comment (cipix2000) [inline tests/jquery.flot.navigate.Test.js]: remove comment

File: tests/jquery.flot.navigate.Test.js (hunk 6 above)
Comment (cipix2000) [inline tests/jquery.flot.navigate.Test.js]: move this function to tests/utils/simulate.js
and create 
simulate.touchStart
simulate.touchMove
simulate.touchEnd

and 

simulate.touchDrag that call all three

File: jquery.flot.touchNavigate.js [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+                eventHolder[0].addEventListener("touchend", onDragEnd, false);
+            }
+        }
+
+        function shutdown(plot, eventHolder) {
+            eventHolder[0].removeEventListener("touchstart", onDragStart);
+            eventHolder[0].removeEventListener('touchmove', onDrag);
+            eventHolder[0].removeEventListener('touchend', onDragEnd);
+
+            if (panTimeout) clearTimeout(panTimeout);
```
Comment (cristiingineru) [inline jquery.flot.touchNavigate.js]: Remove this variable and if statement.

**Pull request merged**